src/hyperlight_host/examples/tracing-tracy/README.md

```diff
@@ -0,0 +1,9 @@
+This is an example of using the tracing-tracy tracing-subscriber. When ran, it will generate traces that can be viewed in the tracy profiler.
+
+You can run it with:
+
+```console
+TRACY_NO_EXIT=1 RUST_LOG=trace cargo run --package hyperlight-host --example tracing-tracy --profile release-with-debug
+```
+
+and then the client should show up in the profiler GUI.
```

Comment:
Would be cool to see some images for this in the README :)

Reply:
fixed